test/parallel/test-util-inspect.js

```diff
@@ -1151,4 +1151,9 @@ if (typeof Symbol !== 'undefined') {
 }
 
 assert.doesNotThrow(() => util.inspect(process));
+
+// @@toStringTag
+assert.strictEqual(util.inspect({ [Symbol.toStringTag]: 'a' }),
+                   'Object [a] { [Symbol(Symbol.toStringTag)]: \'a\' }');
+
```

Comment:
Can you add a test like this?

```js
class Foo {
  constructor() {
    this.foo = 'bar';
  }

  get [Symbol.toStringTag]() {
    return this.foo;
  }
}

assert.strictEqual(util.inspect(new Foo()), 'Foo [bar] { foo: \'bar\' }'));
```

Reply:
To be clear, I don’t think this is a good way to write a class, but it should work :)